rules/defaults.yaml

```diff
@@ -62,6 +62,17 @@
 	TimedUpgradeBar:
 		Upgrade: invulnerability
 
+^ChronoDisable:
```

Comment:
You should make that effect production, sws, weapons and stuff like that, they would be abit more useful for now. Also the Alpha effect can be added. It must be already possible.